documentation/Installing-Clear-Containers-on-Fedora-25.md

```diff
@@ -0,0 +1,60 @@
+#Install the Clear Containers runtime on Fedora 25
+
+##Install the Clear Containers runtime
+```
+$ sudo -E dnf config-manager --add-repo
+http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_25/home:clearlinux:preview:clear-containers-2.1.repo
+$ sudo dnf install cc-oci-runtime linux-container
+```
+
+##Install docker 1.12
+```
+$ sudo -E dnf config-manager --add-repo https://yum.dockerproject.org/repo/testing/fedora/25
```

Comment:
According to documentation of docker: https://docs.docker.com/engine/installation/linux/fedora/

This is the URL of the repo, is there any reason to add the testing repo?

2. sudo dnf config-manager \
    --add-repo \
    https://docs.docker.com/engine/installation/linux/repo_files/fedora/docker.repo